homeassistant/strings.json

```diff
@@ -40,7 +40,8 @@
         "ssl": "Uses an SSL certificate",
         "verify_ssl": "Verify SSL certificate",
         "longitude": "Longitude",
-        "latitude": "Latitude"
+        "latitude": "Latitude",
+        "pin": "PIN Code"
```

Comment:
Is this the translation that we want? Why capitalize `PIN` ? It's not an abbreviation?

Reply:
I thought it was an abbreviation, see https://en.wikipedia.org/wiki/Personal_identification_number.

Reply:
I'll stand with @springstan here 😋

Reply:
why not use the common string when it's being used everywhere else in this PR?

Reply:
That is the common string, that I am using everywhere in this PR.

Reply:
sorry, misread what file you were changing

Reply:
No worries :smile: